test/manual/kerberos.test.js

```diff
@@ -97,6 +97,37 @@ describe('Kerberos', function () {
     });
   });
 
+  context('when passsing SERVICE_HOST', function () {
+    context('when the SERVICE_HOST is invalid', function () {
+      const client = new MongoClient(`${krb5Uri}&maxPoolSize=1`, {
+        authMechanismProperties: {
+          SERVICE_HOST: 'example.com'
+        }
+      });
+
+      it('fails to authenticate', function () {
+        return client.connect().catch(e => {
+          expect(e).to.exist;
```

Comment:
should we check for a more specific error here? also, this logic would actually pass if the connect is successful and the catch never executes - we probably want to do something like 
```
let expectedError;
await client.connect().catch(e => { expectedError = e });
if (!e) {
  expect.fail('Expected connect to fail');
}
expect(e).property(message).to.. // some assertion on the error message or error class
```

Reply:
Nice suggestion I like how clean this is, if we can assert on the property existence at least, (message match would be great) then this is pretty short and sweet, and it should catch the case where there is no error.
```js
let expectedError; 
await client.connect().catch(e => { expectedError = e });
expect(expectedError).to.have.property('message', /x/);
```

Reply:
I think this pattern is worth documenting in our `test/readme.md`! 💡

Reply:
Typo and test updated.